internal/framework/service/turnstile/model.go

```diff
@@ -0,0 +1,15 @@
+package turnstile
+
+import "github.com/hashicorp/terraform-plugin-framework/types"
+
+type TurnstileWidgetModel struct {
+	AccountID    types.String `tfsdk:"account_id"`
+	ID           types.String `tfsdk:"id"`
+	Domains      types.Set    `tfsdk:"domains"`
+	Name         types.String `tfsdk:"name"`
+	Secret       types.String `tfsdk:"secret"`
+	Region       types.String `tfsdk:"region"`
+	Mode         types.String `tfsdk:"mode"`
+	BotFightMode types.Bool   `tfsdk:"bot_fight_mode"`
+	OffLabel     types.Bool   `tfsdk:"off_label"`
```

Comment:
```suggestion
	OffLabel     types.Bool   `tfsdk:"offlabel"`
```

I don't think this is related to the API - so whatever convention is used by tf?